src/main/java/org/folio/circulation/domain/ConfigurationRepository.java

```diff
@@ -34,6 +34,11 @@ public ConfigurationRepository(Clients clients) {
     return lookupConfigurations(cqlQueryResult, applySearchSchedulerNoticesLimit());
   }
 
+  public CompletableFuture<Result<Integer>> lookupSessionTimeout() {
+    Result<CqlQuery> cqlQueryResult = defineModuleNameAndConfigNameFilter("CHECKOUT", "other_settings");
+    return lookupConfigurations(cqlQueryResult, applySessionTimeout());
```

Comment:
Personally I find it easier to read these two lines with a blank line between them

Reply:
The blank has been added